### PR TITLE
Disallow console.log in code  issue #49

### PR DESCRIPTION
**Pull request by albinsabu2023**
#49 
***disabled console.log() method in the source code***

```js
  console.log()
``` 
whenever this method is used in code , red warning with custom message will pop up
here i have given the message  
 ```Unexpected use of console.log```

@StyleShit  kindly review this and comment if any change needed

### CODE REVIEW

File: eslint.config.js [hunk 1]
```diff
@@ -3,26 +3,36 @@ import eslint from '@eslint/js';
...
+    },
+    rules: {
+		"no-console":"off",
+      "no-restricted-syntax": [
+        "error",
+        {
+          "selector": "CallExpression[callee.object.name='console'][callee.property.name='log']",
+          "message": "Unexpected use of console.log."
+        }
+      ]
```
Comment (StyleShit) [inline eslint.config.js:27]: let's simplify the things and use the builtin `no-console` rule

File: eslint.config.js
Comment (StyleShit) [inline eslint.config.js:1]: please use the original file indentation to have less diff

**Review by StyleShit — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Just requesting 2 little changes :)

**Comment by albinsabu2023** [comment]
# simplified the console rule

***added no-console rule  to code***

#49 

@StyleShit  hey i made the changes

File: eslint.config.js [hunk 2]
```diff
@@ -3,26 +3,31 @@ import eslint from '@eslint/js';
...
+    ignores: [
+      '**/coverage/**',
+      '**/dist/**',
+      '**/node_modules/**',
+      '**/__snapshots__/**',
+    ],
+  },
+  {
+    rules:{
+      'no-console':['error':{allow:['warn']}]
```
Comment (StyleShit) [inline eslint.config.js:30]:
```suggestion
      'no-console':['error':{allow:['warn']}]
```

let's disallow any console usage
and please fix the indentation to match the original

**Review by StyleShit — CHANGES_REQUESTED** [review]

File: eslint.config.js [hunk 3]
```diff
@@ -25,4 +25,9 @@ export default tseslint.config(
 			'**/__snapshots__/**',
 		],
 	},
+	{
+		rules:{
+			'no-console':'error'
+		}
+	},
```
Comment (StyleShit) [inline eslint.config.js:32]: Almost there! 🚀 

just one last change - can you please put it together with the `ignores` key?

**Review by StyleShit — CHANGES_REQUESTED** [review]

**Comment by StyleShit** [comment]
and please see the failing checks in the CI

**Comment by albinsabu2023** [comment]
> and please see the failing checks in the CI

ok bro  , but i didn't understand how this rules to put together with ignores key

need a help in that if you dont mind

**Comment by StyleShit** [comment]
not sure that I understand what you mean
but for the CI failures, just run `npm run format` and it should be fixed

**Comment by albinsabu2023** [comment]
> not sure that I understand what you mean but for the CI failures, just run `npm run format` and it should be fixed

{
		ignores: [
			'**/coverage/**',
			'**/dist/**',
			'**/node_modules/**',
			'**/__snapshots__/**',
		],
		rules: {
			'no-console': 'error',
		},
	},

is that what you mean ,yeah i will run the prettier format to avoid conflicts

**Comment by StyleShit** [comment]
yup :)

**Review by StyleShit — APPROVED** [review]
LGTM

Thanks! 🔥

**Comment by albinsabu2023** [comment]
> LGTM
> 
> Thanks! 🔥

thankyou for your guidance

**Pull request merged**